client/src/components/Viewer/components/GetFiles/components/RenderFiles/components/OpenSeaDragon/OpenSeadragonViewer.js

```diff
@@ -1,103 +1,75 @@
 import React, { useEffect } from 'react';
 import OpenSeadragon from 'openseadragon';
 import './OpenSeadragon.css';
+import { config } from "../../../../../../../Config/config";
```

Comment:
dont use relative path. use absolute wrt index.js

Reply:
Added jsconfig.json in client folder to configure the baseURL = 'src' for absolute paths.